runelite-client/src/main/java/net/runelite/client/config/ConfigManager.java

```diff
@@ -179,11 +183,11 @@ private synchronized void loadFromFile()
 
 		try (FileInputStream in = new FileInputStream(propertiesFile))
 		{
-			properties.load(in);
+			properties.load(new InputStreamReader(in, Charset.forName("UTF-8")));
 		}
 		catch (FileNotFoundException ex)
 		{
-			log.debug("Unable to load settings - no such file");
+			log.debug("Unable to load settings - no such file", ex);
```

Comment:
exception here was intentionally not logged since this is part of normal operation